aybapi/impl/category.py

```diff
@@ -0,0 +1,26 @@
+from ..utils.errors import *
+
+
+class Category:
+    def __init__(self, info):
+        self.success = info['success']
+
+        if (self.success):
+            self.id = info['id']
+            self.name = info['name']
+            self.slug = info['slug']
+        else:
+
+            raise APIError(info['note'])
+
+    def __str__(self):
```

Comment:
this is a `repr` really, `str` should be a simple view, like the name only usually.